deploy/images/db/grants.sql

```diff
@@ -13,15 +13,10 @@ CREATE USER lega_in;
 CREATE USER lega_out;
 
 -- Set up rights access for local_ega schema
-GRANT USAGE ON SCHEMA local_ega TO lega_in, lega_out;
+GRANT USAGE ON SCHEMA local_ega TO lega_in;
```

Comment:
Are you sure lega_out does not need usage on local_ega schema in order to see the main table?